packages/core-utils/src/types.js

```diff
@@ -603,3 +603,20 @@ export const userLocationType = PropTypes.shape({
    */
   type: PropTypes.string.isRequired
 });
+
+/**
+ * The type for each option in the queryParams object.
+ */
+export const queryParamOptionType = PropTypes.shape({
+  text: PropTypes.string.isRequired,
+  value: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.string,
+    PropTypes.bool
+  ]).isRequired
+});
```

Comment:
As I mentioned in #274, I'm not too crazy about adding new PropTypes. If a PR is adding new functionality, we should probably be adding typescript to the files we touch.

Reply:
Removed in https://github.com/opentripplanner/otp-ui/pull/273/commits/e3639093dad1357e7d0ab2a84afaf7e779d52c0d ion favor of adding TypeScript in the corresponding component package in a separate PR.